ui/web/package.json

```diff
@@ -134,6 +133,7 @@
     "less-loader": "^5.0.0",
     "recompose": "^0.30.0",
     "redux-devtools-extension": "^2.13.8",
+    "reduxsauce": "^1.1.0",
```

Comment:
> We're happy you decided to check this project out. Since we no longer use it at Infinite Red, we've decided to archive it. So it's no longer maintained.

https://github.com/infinitered/reduxsauce

:(

Reply:
Do we have any alternatives we could use?

Reply:
(┛◉Д◉)┛彡┻━┻

We could look into alternatives in the future but I don't find this to be an immediate issue as we are rapidly iterating on the UI and the related state management patterns anyways. Whether we even continue to use Redux is always a consideration.